homeassistant/components/template/lock.py

```diff
@@ -198,6 +204,20 @@ async def async_unlock(self, **kwargs: Any) -> None:
             self._command_unlock, run_variables=tpl_vars, context=self._context
         )
 
+    async def async_open(self, **kwargs: Any) -> None:
```

Comment:
Please add the `is_open` property too otherwise, the state will never be opened

Reply:
`is_open` property was added now.